### PR TITLE
OSDOCS-11897_rn#release note IAM instance profile

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Version:
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-11897

Link to docs preview:
https://81949--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#installing-a-cluster-on-amazon-web-services-using-an-existing-iam-profile

QE review:
- [x] QE has approved this change.

Additional information:
Doc PR: https://github.com/openshift/openshift-docs/pull/81715

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -55,6 +55,10 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-4-17-installation-and-update_{context}"]
 === Installation and update
 
+==== Installing a cluster on {aws-full} using an existing IAM profile
+
+With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. You can specify either an existing IAM profile by using the `iamRole` parameter or you can specify a new IAM profile by using the new `iamProfile` parameter, but you cannot specify both. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```
Comment (yunjiang29) [inline release_notes/ocp-4-17-release-notes.adoc:60]: I don't think we need to mention `iamRole` in the release note, as we already have a clear description in the [parameter table](https://github.com/openshift/openshift-docs/pull/81715)

```suggestion
With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -81,6 +81,9 @@ With the replacement of Terraform, the following permissions are required if you
 
 For more information on required permissions, see xref:../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure].
 ====
+==== Installing a cluster on {aws-full} using an existing IAM profile
```
Comment (xenolinux) [inline release_notes/ocp-4-17-release-notes.adoc:84]:
```suggestion
==== Installing a cluster on {aws-short} by using an existing IAM profile
```
I would suggest to consider using "AWS" in the heading and the full form in the paragraph. This also makes a heading smaller.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -81,6 +81,9 @@ With the replacement of Terraform, the following permissions are required if you
...
+
+With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```
Comment (xenolinux) [inline release_notes/ocp-4-17-release-notes.adoc:86]:
```suggestion
With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
```

**Review by xenolinux — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

One small suggestion; otherwise LGTM!

**Pull request merged**